ethers-core/src/types/chain.rs

```diff
@@ -299,6 +303,9 @@ impl Chain {
             Chiado => {
                 ("https://blockscout.chiadochain.net/api", "https://blockscout.chiadochain.net")
             }
+            FilecoinHyperspaceTestnet => {
+                ("https://api.hyperspace.node.glif.io/rpc/v1", "https://hyperspace.filfox.info")
+            }
```

Comment:
are these compatible with etherscan API?

Reply:
They are not

Reply:
Hmm then we cannot use them. Where can we read more about the filecoin evm spec?

Reply:
@gakonst  Here are some resources regarding docs/specs:
- [Spec](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0030.md)
- [Official docs](https://docs.filecoin.io/developers/smart-contracts/filecoin-virtual-machine/)